examples/impute/plot_multiple_imputation.py

```diff
@@ -0,0 +1,457 @@
+                                   sample_posterior=True,
+                                   random_state=i)
+        imputer.fit(Xy)
+        data_imputed = imputer.transform(Xy)
+
+        # We save only the X imputed data because we do not want to use y to
+        # predict y later on.
+        X_imputed = data_imputed[:, :-1]
+        multiple_imputations.append(X_imputed)
+
+    # Perform linear regression on mice multiple imputed data
+    # Estimate beta estimates and their variances
+    m_coefs = []
+    m_vars = []
+    for i in range(m):
+        estimator = RidgeCV()
+        estimator.fit(multiple_imputations[i], y)
+        y_predict = estimator.predict(multiple_imputations[i])
+        m_coefs.append(estimator.coef_)
+        m_vars.append(calculate_variance_of_beta_estimates(
+                y, y_predict, multiple_imputations[i]))
+
+    # Calculate the end estimates by applying Rubin's rules.
+    Qbar = calculate_Qbar(m_coefs)
+    T = calculate_T(m_coefs, m_vars, Qbar)
+    mice_errorbar = 1.96 * np.sqrt(T)
+
+    return Qbar, T, mice_errorbar
+
+
+# Now lets run all these imputation procedures.
+# We use the Boston dataset and analyze the outcomes of the beta coefficients
+# and their standard errors. We standardize the data before running the
+# procedure to be able to compare the coefficients. We run the procedure for
+# MCAR missingness only.
+#
+# Loading the data
+dataset = load_boston()
+X_full, y = dataset.data, dataset.target
+
+# Standardizing the data
+scaler = StandardScaler()
+X_scaled = scaler.fit_transform(X_full)
+y_scaled = stats.zscore(y)
+
+# Start the procedure
+print("Executing Example 1 MCAR Missingness...")
+
+# First, make the data incomplete with a MCAR mechanism.
+X_incomplete = ampute(X_scaled, mech="MCAR")
+
+# Second, run all the imputation procedures as described above.
+full_coefs, full_vars, full_errorbar = get_results_full_dataset(
+        X_scaled, y_scaled)
+chained_coefs, chained_vars, chained_errorbar = get_results_chained_imputation(
+        X_incomplete, y_scaled)
+mice_coefs, mice_vars, mice_errorbar = get_results_mice_imputation(
+        X_incomplete, y_scaled)
+mice_y_coefs, mice_y_vars, mice_y_errorbar = \
+    get_results_mice_imputation_includingy(
+        X_incomplete, y_scaled)
+
+# Combine the results from the four imputation procedures.
+coefs = (full_coefs, chained_coefs, mice_coefs, mice_y_coefs)
+vars = (full_vars, chained_vars, mice_vars, mice_y_vars)
+errorbars = (full_errorbar, chained_errorbar, mice_errorbar, mice_y_errorbar)
+
+# And plot the results
+n_situations = 4
+n = np.arange(n_situations)
+n_labels = ['Full Data', 'IterativeImputer',
+            'Mice Imputer', 'Mice Imputer with y']
+colors = ['r', 'orange', 'b', 'purple']
+width = 0.3
+plt.figure(figsize=(24, 32))
+
+plt1 = plt.subplot(211)
+for j in n:
+    plt1.bar(np.arange(len(coefs[j])) + (3*j*(width/n_situations)),
+             coefs[j], width=width, color=colors[j])
```

Comment:
I think there must be some confusion due to the variable naming. The two figures that are part of example 1 do not have error bars because they are the result of a single run. `errorbars` are actually the standard errors.